doc/sources/quick-start.rst

```diff
@@ -14,17 +14,455 @@
 .. * limitations under the License.
 .. *******************************************************************************/
 
-.. _quick_start:
+.. |intelex_repo| replace:: |intelex| repository
+.. _intelex_repo: https://github.com/intel/scikit-learn-intelex
 
-############################################
+####################
 Quick Start
-############################################
+####################
+
+Get ready to elevate your scikit-learn code with |intelex| and experience the benefits of accelerated performance in just a few simple steps. 
+
+Compatibility with Scikit-learn*
+---------------------------------
+
+Intel(R) Extension for Scikit-learn is compatible with the following versions of scikit-learn:
+
+* 1.0.X
```

Comment:
Spending lot of space with not enough value - can we make this more compact?  In general i would suggest just saying that 4 last release scikit versions are supported here and will link for exact details to system requirements

Reply:
fix